person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'unkwomn', parent_permission: true)
+    @id = Random.rand(1..1000),
+          @name = name,
+          @age = age,
+          @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
+
+  private
+
+  def can_use_services?
+    if is_of_age? || parent_permission
```

Comment:
- [ ] It would be better if we check the `is_of_age?` and `of_age` and solve it. Otherwise, line 19 will return "method not defined"